pkg/networking/subnet_resolver.go

```diff
@@ -48,6 +48,8 @@ type SubnetsResolveOptions struct {
 	AvailableIPAddressCount int64
 	// whether to check the cluster tag
 	SubnetsClusterTagCheck bool
+	// Disable subnet minimal count restriction
```

Comment:
Maybe reword: whether to allow using only 1 subnet for provisioning ALB, default to `false`